src/ServicePulse.Host/app/modules/monitoring/views/monitored_endpoints.html

```diff
@@ -81,9 +81,14 @@ <h1>Endpoints overview</h1>
                                         <span class="warning" ng-if="endpoint.isScMonitoringDisconnected">
                                             <i class="fa pa-monitoring-lost endpoints-overview" uib-tooltip="Unable to connect to monitoring server"></i>
                                         </span>
-                                        <span class="warning" ng-if="endpoint.isStale">
+                                        <span class="warning" ng-if="endpoint.isStale && (endpoint.connectedCount==null || endpoint.connectedCount==0)">
```

Comment:
```suggestion
                                        <span class="warning" ng-if="endpoint.isStale && endpoint.connectedCount">
```

Type coalescing in JavaScript will convert the check to:

* Does endpoint.connectedCount have a defined value?
* Is the value non-null?
* Is the value non-zero?
* Is the value a non-empty string?